<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WelcomePage</class>
 <widget class="QWidget" name="WelcomePage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1104</width>
    <height>809</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:transparent;</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>13</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QStackedWidget" name="stackedWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="welcomePage">
      <layout class="QVBoxLayout" name="verticalLayout_4" stretch="1,1,7,0,0">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Expanding</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QWidget" name="widget" native="true">
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QLabel" name="label">
            <property name="font">
             <font>
              <pointsize>22</pointsize>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">color:rgb(78, 78, 78);</string>
            </property>
            <property name="text">
             <string>欢迎使用可瑞!</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="widget_2" native="true">
         <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="0,1,1,0">
          <property name="spacing">
           <number>4</number>
          </property>
          <property name="topMargin">
           <number>10</number>
          </property>
          <item>
           <spacer name="horizontalSpacer_4">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QWidget" name="widget_3" native="true">
            <layout class="QVBoxLayout" name="verticalLayout_2">
             <item>
              <widget class="QWidget" name="widget_5" native="true">
               <widget class="QWidget" name="widget_7" native="true">
                <property name="geometry">
                 <rect>
                  <x>130</x>
                  <y>20</y>
                  <width>172</width>
                  <height>211</height>
                 </rect>
                </property>
                <property name="minimumSize">
                 <size>
                  <width>172</width>
                  <height>171</height>
                 </size>
                </property>
                <widget class="QLabel" name="label_2">
                 <property name="geometry">
                  <rect>
                   <x>30</x>
                   <y>-5</y>
                   <width>61</width>
                   <height>41</height>
                  </rect>
                 </property>
                 <property name="font">
                  <font>
                   <pointsize>14</pointsize>
                  </font>
                 </property>
                 <property name="styleSheet">
                  <string notr="true">color:rgb(78, 78, 78);</string>
                 </property>
                 <property name="text">
                  <string>开始</string>
                 </property>
                </widget>
                <widget class="QPushButton" name="newPage">
                 <property name="geometry">
                  <rect>
                   <x>0</x>
                   <y>50</y>
                   <width>161</width>
                   <height>31</height>
                  </rect>
                 </property>
                 <property name="font">
                  <font>
                   <pointsize>12</pointsize>
                  </font>
                 </property>
                 <property name="cursor">
                  <cursorShape>PointingHandCursor</cursorShape>
                 </property>
                 <property name="styleSheet">
                  <string notr="true">color:rgb(90, 166, 255);</string>
                 </property>
                 <property name="text">
                  <string>新建项目</string>
                 </property>
                 <property name="icon">
                  <iconset resource="images.qrc">
                   <normaloff>:/welcomePage/images/welcomePage/newPage.svg</normaloff>:/welcomePage/images/welcomePage/newPage.svg</iconset>
                 </property>
                 <property name="iconSize">
                  <size>
                   <width>35</width>
                   <height>35</height>
                  </size>
                 </property>
                 <property name="flat">
                  <bool>true</bool>
                 </property>
                </widget>
                <widget class="QPushButton" name="selectFolder">
                 <property name="geometry">
                  <rect>
                   <x>0</x>
                   <y>110</y>
                   <width>161</width>
                   <height>31</height>
                  </rect>
                 </property>
                 <property name="font">
                  <font>
                   <pointsize>12</pointsize>
                  </font>
                 </property>
                 <property name="cursor">
                  <cursorShape>PointingHandCursor</cursorShape>
                 </property>
                 <property name="styleSheet">
                  <string notr="true">color:rgb(90, 166, 255);</string>
                 </property>
                 <property name="text">
                  <string>选择项目</string>
                 </property>
                 <property name="icon">
                  <iconset resource="images.qrc">
                   <normaloff>:/welcomePage/images/welcomePage/selectFloder.svg</normaloff>:/welcomePage/images/welcomePage/selectFloder.svg</iconset>
                 </property>
                 <property name="iconSize">
                  <size>
                   <width>35</width>
                   <height>35</height>
                  </size>
                 </property>
                 <property name="flat">
                  <bool>true</bool>
                 </property>
                </widget>
               </widget>
               <widget class="QWidget" name="widget_6" native="true">
                <property name="geometry">
                 <rect>
                  <x>-10</x>
                  <y>260</y>
                  <width>501</width>
                  <height>371</height>
                 </rect>
                </property>
                <widget class="QWidget" name="widget_9" native="true">
                 <property name="geometry">
                  <rect>
                   <x>140</x>
                   <y>20</y>
                   <width>201</width>
                   <height>241</height>
                  </rect>
                 </property>
                 <widget class="QLabel" name="label_3">
                  <property name="geometry">
                   <rect>
                    <x>30</x>
                    <y>10</y>
                    <width>61</width>
                    <height>41</height>
                   </rect>
                  </property>
                  <property name="font">
                   <font>
                    <pointsize>14</pointsize>
                   </font>
                  </property>
                  <property name="styleSheet">
                   <string notr="true">color:rgb(78, 78, 78);</string>
                  </property>
                  <property name="text">
                   <string>了解</string>
                  </property>
                 </widget>
                 <widget class="QPushButton" name="pushButton_3">
                  <property name="geometry">
                   <rect>
                    <x>10</x>
                    <y>80</y>
                    <width>181</width>
                    <height>31</height>
                   </rect>
                  </property>
                  <property name="font">
                   <font>
                    <pointsize>12</pointsize>
                   </font>
                  </property>
                  <property name="cursor">
                   <cursorShape>PointingHandCursor</cursorShape>
                  </property>
                  <property name="styleSheet">
                   <string notr="true">color:rgb(90, 166, 255);</string>
                  </property>
                  <property name="text">
                   <string>如何使用可瑞？</string>
                  </property>
                  <property name="iconSize">
                   <size>
                    <width>40</width>
                    <height>40</height>
                   </size>
                  </property>
                  <property name="flat">
                   <bool>true</bool>
                  </property>
                 </widget>
                 <widget class="QPushButton" name="pushButton_4">
                  <property name="geometry">
                   <rect>
                    <x>10</x>
                    <y>130</y>
                    <width>201</width>
                    <height>31</height>
                   </rect>
                  </property>
                  <property name="font">
                   <font>
                    <pointsize>12</pointsize>
                   </font>
                  </property>
                  <property name="cursor">
                   <cursorShape>PointingHandCursor</cursorShape>
                  </property>
                  <property name="styleSheet">
                   <string notr="true">color:rgb(90, 166, 255);</string>
                  </property>
                  <property name="text">
                   <string>了解编程基础知识</string>
                  </property>
                  <property name="iconSize">
                   <size>
                    <width>40</width>
                    <height>40</height>
                   </size>
                  </property>
                  <property name="flat">
                   <bool>true</bool>
                  </property>
                 </widget>
                </widget>
               </widget>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QWidget" name="widget_4" native="true">
            <layout class="QVBoxLayout" name="verticalLayout_3" stretch="1,9,1">
             <item>
              <widget class="QWidget" name="widget_8" native="true">
               <widget class="QLabel" name="label_4">
                <property name="geometry">
                 <rect>
                  <x>10</x>
                  <y>10</y>
                  <width>61</width>
                  <height>41</height>
                 </rect>
                </property>
                <property name="minimumSize">
                 <size>
                  <width>61</width>
                  <height>41</height>
                 </size>
                </property>
                <property name="font">
                 <font>
                  <pointsize>14</pointsize>
                 </font>
                </property>
                <property name="styleSheet">
                 <string notr="true">color:rgb(78, 78, 78);</string>
                </property>
                <property name="text">
                 <string>最近</string>
                </property>
               </widget>
              </widget>
             </item>
             <item>
              <widget class="QScrollArea" name="scrollArea">
               <property name="maximumSize">
                <size>
                 <width>600</width>
                 <height>600</height>
                </size>
               </property>
               <property name="styleSheet">
                <string notr="true">#scrollArea{
border:0;
}</string>
               </property>
               <property name="verticalScrollBarPolicy">
                <enum>Qt::ScrollBarAlwaysOff</enum>
               </property>
               <property name="horizontalScrollBarPolicy">
                <enum>Qt::ScrollBarAlwaysOff</enum>
               </property>
               <property name="widgetResizable">
                <bool>true</bool>
               </property>
               <widget class="QWidget" name="recentlyProject_scrollArea">
                <property name="geometry">
                 <rect>
                  <x>0</x>
                  <y>0</y>
                  <width>473</width>
                  <height>392</height>
                 </rect>
                </property>
                <property name="styleSheet">
                 <string notr="true">border:0;</string>
                </property>
                <layout class="QVBoxLayout" name="verticalLayout_5">
                 <item>
                  <layout class="QVBoxLayout" name="recentlyProject_layout">
                   <property name="spacing">
                    <number>25</number>
                   </property>
                  </layout>
                 </item>
                </layout>
               </widget>
              </widget>
             </item>
             <item>
              <spacer name="verticalSpacer_3">
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>20</width>
                 <height>40</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_3">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Expanding</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QWidget" name="widget_10" native="true">
         <layout class="QVBoxLayout" name="verticalLayout_6">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QWidget" name="BottomTabBar" native="true">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>26</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>26</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">background-color: rgb(255, 216, 180);</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="analyzingPage"/>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="images.qrc"/>
 </resources>
 <connections/>
</ui>
